config/locales/exceptions.en.yml

```diff
@@ -1051,12 +1039,6 @@ en:
         code: VET360_ADDR300
         detail: Cannot modify an existing inactive record.
         status: 400
-      VET360_ADDR300:
-        <<: *external_defaults
-        title: Address Inactive
-        code: VET360_ADDR300
-        detail: Cannot modify an existing inactive record.
-        status: 400
```

Comment:
accidental line deletes?

Reply:
Nope, I believe these were all duplicates. Had to tell in the github minified change view.